tutorials/milvus_lite_arize.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>Instead of "Create a vector store from a json file and set it up with Milvus Lite", something like "Load a pre-built knowledge into a Milvus Lite vector store". There probably needs to be some discussion of what Milvus + Milvus Lite are.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>